app/controllers/multipage_contents_controller.rb

```diff
@@ -0,0 +1,31 @@
+class MultipageContentsController < ApplicationController
+  def show
+    content_item_response = ContentStore.service.content_item(base_path)
+    @content = model_class.new(content_item_response.to_hash) if content_item_response
+
+    if @content.present?
+      if params[:part]
+        @current_part = current_part
+        redirect_to base_path unless @current_part
+      end
+    else
+      render_404
+    end
+  end
```

Comment:
No, the idea (taken from [alphagov/frontend](https://github.com/alphagov/frontend/blob/master/app/controllers/travel_advice_controller.rb#L51..L69) is that unless the current part exists on the content item we are redirected to the base country path. Only of the content item doesn't exist do we 404. So `/foreign-travel-advice/legoland` is a `404` whereas `/foreign-travel-advice/spain/part-does-not-exist` redirects to '/foreign-travel-advice/spain'.
[Tests here](https://github.com/alphagov/multipage-frontend/commit/295456d4eecd95c4d9acad07f2286eea7b92e23c#diff-794b79da3239792939aa3993f60355abR27) cover these 3 scenarios.